src/workers/userUpdatedCio.ts

```diff
@@ -0,0 +1,54 @@
+import { getFirstName, ghostUser } from '../common';
+import { TypedWorker } from './worker';
+import { cio, dateToCioTimestamp } from '../cio';
+import { User } from '../entity';
+
+function camelCaseToSnakeCase(
+  obj: Record<string, unknown>,
+): Record<string, unknown> {
+  const snakeObj: Record<string, unknown> = {};
+  for (const key in obj) {
+    snakeObj[key.replace(/[A-Z]/g, (letter) => `_${letter.toLowerCase()}`)] =
+      obj[key];
+  }
+  return snakeObj;
+}
+
+const OMIT_FIELDS: (keyof User)[] = [
+  'id',
+  'bio',
+  'devcardEligible',
+  'readme',
+  'readmeHtml',
+  'infoConfirmed',
+  'profileConfirmed',
+  'notificationEmail',
+];
+
+const worker: TypedWorker<'user-updated'> = {
```

Comment:
Just for updates? Or does it also run on user created?

Reply:
originally i thought that on creation a user can't be confirmed but i think with email they can. i'll double check